File: 100-Days-Series/Day 64/CheckerboardPattern.py

# Function to print Checkerboard Pattern
def get_print_chekerboard(n):
    for i in range(1,n+1,1):
        for j in range(1,n+1,1):
            if (i+j)%2==0:
                print(end="* ")
            else:
                print(end="  ")
        print()


# Let's Start Here
n=int(input("Enter the size of Chekerboard : "))
get_print_chekerboard(n)

File: 100-Days-Series/Day 64/CheckerboardPattern.cpp
#include<iostream>
using namespace std;
// Function to print Checkerboard Pattern5
void getPrintCheckerboard(int n){
    for(int i=1;i<=n;i++){
        for(int j=1;j<=n;j++){
            if((i+j)%2==0){
                cout<<"* ";
            }else {
                cout<<"  ";
            }
        }
        cout<<endl;
    }
}
// Let's Start Here
int main(){
    int n;
    cout<<"\nEnter the size of Checkerboard : ";
    cin>>n;
    getPrintCheckerboard(n);
}